docs/wiki/ConversionScript.md

```diff
@@ -0,0 +1,50 @@
+# Convert Bicep files into ARM helper script
+
+As Bicep is still in a beta phase and many people are used to ARM, it may make sense, to wait for Bicep's _General Availability_. In that case, the bicep module files need to be converted to ARM files. A translator/compiler to do that is part of the Bicep toolkit. To run the tool on all bicep files and do some cleanup, a conversion script is supplied with the CARML library. This page is about this script and how to use it.
+
+---
+
+### _Navigation_
+
+- [Location](#location)
+- [What it does](#what-it-does)
+- [How to use it](#how-to-use-it)
+  - [Examples](#examples)
+  
+---
+# Location
+
+`You can find the script under /utilities/tools/ConvertTo-ARMTemplate.ps1`
+
+# What it does
+
+The script finds all 'deploy.bicep' files and tries to convert them to json-based ARM templates
+by using the following steps.
+1. Remove existing deploy.json files
+1. Convert bicep files to json
```

Comment:
I'd suggest to choose between `Bicep `or `bicep ` and use it consistently thoughout the page

Reply:
done